src/main/java/com/jalasoft/selenium/leonardo/movies/Movie.java

```diff
@@ -0,0 +1,45 @@
+package com.jalasoft.selenium.leonardo.movies;
+
+/**
+ *
+ */
+public class Movie extends movieType {
```

Comment:
I think you should have a parent Movie class
and child classes for NewRelease, Children, and Regular that extends from Movie class